pkg/operation/botanist/monitoring.go

```diff
@@ -457,12 +458,20 @@ func (b *Botanist) DeploySeedGrafana(ctx context.Context) error {
 		return kutil.DeleteObject(ctx, b.K8sGardenClient.Client(), &corev1.Secret{ObjectMeta: metav1.ObjectMeta{Name: secretName, Namespace: b.Shoot.GetInfo().Namespace}})
 	}
 
-	credentialsSecret, err := b.SecretsManager.Generate(ctx, observabilityIngressSecretConfig(secretNameIngressOperators), secretsmanager.Persist())
+	operatorsGenerateOptions := []secretsmanager.GenerateOption{
```

Comment:
Do we need these options as separate variable? I would prefer keeping it before and just separating the options directly in the `Generate` call? Same below

Reply:
Maybe you can use newlines to improve readability instead?